### ISSUE TITLE
[BUG] When we switch account, all info like transactions, balance from previous account should disappear and show just loading indicator

### ISSUE THREAD

**Issue by bitcoinuser** [getAlby/lightning-browser-extension#2407]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

Hi,

Hi, when we switch account, all info from previous account like transactions, balance should disappear and show just loading indicator, until the data of the new account load.

### Screenshots [optional]

See the video:

https://user-images.githubusercontent.com/1178856/236932843-cbae740c-a63a-4fa0-b240-992d4790a784.mp4

### Steps To Reproduce

1- You need to have two accounts and one of them should take some time to load
2- Switch account and see that data from previous account (Balance and transactions history) still appear until the new account data load

### Expected behavior

When we switch account, all info from previous account should "disappear" and show just loading indicator until the data of the new account load.

### Alby information

Alby version: 2.0

### Additional context

It's similar to this problem:

https://github.com/getAlby/lightning-browser-extension/issues/2219

### Are you working on this?

No

**Comment by reneaaron** [comment]
Thanks for your feedback, we are aware of this problem and will work on a fix. As we'll have to refactor some code that does the account & balance loading that could take some time, please bear with us till then.

Related to #2377

**Comment by lujakob** [comment]
@reneaaron I started debugging and refactor to have an explicit "accountLoading" state to show a loading state on account change where needed. I already have a working draft, so you could assign me this ticket. I would like to refactor and unify wallet page and DefaultView as you suggested first, and once this is merged into master build upon it. I will show you the draft/implementation then.

Also I'd suggest to address "error handling" #2377 and this improved "loading state" in two separate PR's.
